service/src/main/java/org/dice/FactCheck/Corraborative/PathGenerator/WIKIDataPathGenerator.java

```diff
@@ -0,0 +1,151 @@
+package org.dice.FactCheck.Corraborative.PathGenerator;
+
+import java.util.HashMap;
+import org.apache.jena.query.ParameterizedSparqlString;
+import org.apache.jena.query.QueryExecution;
+import org.apache.jena.query.QuerySolution;
+import org.apache.jena.query.ResultSet;
+import org.apache.jena.rdf.model.Statement;
+import org.dice.FactCheck.Corraborative.PathQuery;
+import org.dice.FactCheck.Corraborative.Query.QueryExecutioner;
+
+public class WIKIDataPathGenerator implements IPathGenerator {
```

Comment:
It is written `Wikidata` (see https://www.wikidata.org/wiki/Wikidata:Main_Page) :wink: